src/webhooks/__tests__/registry.test.ts

```diff
@@ -54,10 +54,10 @@ async function genericWebhookHandler(
 
 const TEST_WEBHOOK_ID = 'gid://shopify/WebhookSubscription/12345';
 
-describe('ShopifyWebhooks.Registry.register', () => {
+describe('shopify.webhooks.registry.register', () => {
   beforeEach(async () => {
-    config.apiVersion = ApiVersion.Unstable;
-    ShopifyWebhooks.Registry.webhookRegistry = {};
+    shopify.config.apiVersion = ApiVersion.Unstable;
+    shopify.webhooks.registry.webhookRegistry = {};
```

Comment:
If we're creating a new instance for each test, do we still need to do this? Just curious 🙂